runtime/caml/startup_aux.h

```diff
@@ -58,6 +59,11 @@ extern const struct caml_params* const caml_params;
 
 extern void caml_parse_ocamlrunparam (void);
 
+#if !defined(CAML_INTERNALS_NO_PRIM_DECLARATIONS)
+CAMLextern value caml_hashtbl_is_randomized(value vunit);
+CAMLextern value caml_hashtbl_randomize(value vunit);
```

Comment:
I'm probably missing something, but why are these here at all?

Reply:
You're right, the primitives don't need to be exposed.